### ISSUE TITLE
[Web UI] Support `scenarios` yaml config option

### ISSUE THREAD

**Issue by chrismaltais** [promptfoo/promptfoo#1068]
https://github.com/promptfoo/promptfoo/assets/17098959/38227a4a-9501-48a9-b527-6ce1a65959b5

I have the following YAML config file that defines the eval I want to run. I want to be able to run this via the Web UI:
```yaml
description: 'Test'
prompts:
- file://src/providers/test/prompt.txt
providers:
- id: 'python:src/providers/test/openai/translate.py'
  label: 'GPT-4o'
  config:
    model: 'gpt-4o'
- id: 'python:src/providers/test/openai/translate.py'
  label: 'GPT-3.5-Turbo'
  config:
    model: 'gpt-3.5-turbo'

scenarios:
- config:
  - description: English
    vars:
      language: en (English)
  - description: Spanish
    vars:
      language: es (Spanish)
  - description: French
    vars:
      language: fr (French)
  - description: German
    vars:
      language: de (German)

  tests:
  - vars:
      tone: Expert
  - vars:
      tone: Daring
  - vars:
      tone: Persuasive
  - vars:
      tone: Playful
  - vars:
      tone: Sophisticated
  - vars:
      tone: Supportive
```

I don't think the `scenarios` field gets saved, which would happen [here](https://github.com/promptfoo/promptfoo/blob/b4267a03b392f386ce95be6e43565d12bcab97c8/src/web/nextui/src/app/setup/YamlEditor.tsx#L39-L45)

**Comment by mldangelo** [comment]
Hi @chrismaltais, Thanks for reporting this issue and providing detailed information! We've identified the problem with the `scenarios` field not being saved. A fix is currently on the way.  I'll update you once the fix is deployed. Your feedback is invaluable in helping us improve the Web UI. Thanks again!

**Comment by typpo** [comment]
This is published in 0.68.2.
